src/app/child-dev-project/notes/note-details/note-config.interface.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Object specifying overall note
+ * as stored in the config database
+ */
+export interface NoteConfig {
+  InteractionTypes: {
+    [key: string]: {
```

Comment:
do we really still need the `key` value format? I think it might be easier to simply have an array of available interaction type object, each with name, color, etc.?